src/utils/systemnotification.cpp

```diff
@@ -20,7 +20,7 @@ SystemNotification::SystemNotification(QObject *parent) : QObject(parent) {
 }
 #else
 SystemNotification::SystemNotification(QObject *parent) : QObject(parent) {
-
+    m_interface = null;
```

Comment:
ppveyor's build fails because of this, it should be nullptr instead.

Reply:
@lupoDharkael Yep, I totally missed that. I fixed this in my local branch and added `cppcheck`, I'm testing in a local branch.